Jenkinsfile

```diff
@@ -84,6 +84,12 @@ pipeline {
                             'CAUTION: only use in combination with a reduced ' +
                             'number of tests specified with the TestTag ' +
                             'parameter.')
+        string(name: 'TestProvider',
+               defaultValue: "",
```

Comment:
FYI: Groovy prefers single quotes for strings that don't have any variable interpolation in them.